headerdetection.py

```diff
@@ -0,0 +1,32 @@
+import cv2
+def headerdetection(g,lowerlimit,upperlimit,dd,b,img):
+    headerrownumber=[]
```

Comment:
define what this function does. Add comments everywhere you are doing manipulations.

Reply:
For functions I can write a blog along with images...for different threshold I have explained using comments